### ISSUE TITLE
aws configure & IAM 

### ISSUE THREAD

**Issue by jayyeras** [mandusm/aws-modernization-workshop#79]
`aws configure` instructions are not found in Getting Started module. I also do not see IAM access keys to proceed with configuration. Currently blocked on Artifactory CloudFormation stack until resolved.

**Comment by jayyeras** [comment]
Per @mandusm modules should be using instance roles, but "no default region" error is based on not having configured AWS DEFAULT REGION. Closing this as it is already filed under separate issue.
